vision/benchmarks/accuracy/bench_module.py

```diff
@@ -164,8 +164,42 @@ def accuracy_get_module_roll(self, region: np.ndarray) -> float:
         """
         return get_module_roll(region)
 
+    def accuracy_saveCircleImage(
+        self, filename, color_image: np.ndarray, depth_image: np.ndarray
+    ) -> None:
+        """
+        Creates a folder and calls on the saveCircleImage function to save images with circles.
+
+        Parameters
+        ----------
+        color_image: ndarray
+            The color image.
+        depth_image: ndarray
+            The depth image.
+        filename
+            The file name.
+
+        Returns
+        -------
+        None
+        """
+        OUTPUT_FILE = "Circle_Images"
+        try:
+            os.mkdir(OUTPUT_FILE)
+        except FileExistsError:
+            pass
+        self.location.setImg(color_image, depth_image)
+        self.location.getCenter()
```

Comment:
Also, since this is called after getCenter has already been called for the image, you don't need to set the image or getCenter() again. You only need the line after this that saves the circle.